### DISCUSSION TITLE
1.12.13.0 ctd

### DISCUSSION THREAD

**Discussion by Mickg56** [flybywiresim/aircraft#2618] [Help]
Hi guys...just discovered FS 2020 ctd using latest flybywire a32nx. Any news?

**Comment by dellycowboy** [comment]
So I can recreate this crash with stable and dev versions. 

It only happens when starting a flight from the runway, from a gate is unaffected.

As the flight is about to load, I hear sounds then the Captain starts to say "seats for take-off" and I get CTD with 2 Event Log Errors

1. 

Faulting application name: AUDIODG.EXE, version: 10.0.18362.1256, time stamp: 0xdda6cbad
Faulting module name: ntdll.dll, version: 10.0.18362.1171, time stamp: 0x103a4719
Exception code: 0xc0000005
Fault offset: 0x00000000000a01fd
Faulting process ID: 0x5b40
Faulting application start time: 0x01d6d8ac8750cf49
Faulting application path: C:\Windows\system32\AUDIODG.EXE
Faulting module path: C:\Windows\SYSTEM32\ntdll.dll
Report ID: 35dbb574-7f9d-42e9-b278-308e49ee40fb
Faulting package full name: 
Faulting package-relative application ID: 

2. 

Faulting application name: FlightSimulator.exe, version: 1.12.13.0, time stamp: 0x5fda3fe4
Faulting module name: FlightSimulator.exe, version: 1.12.13.0, time stamp: 0x5fda3fe4
Exception code: 0xc0000005
Fault offset: 0x000000000190870a
Faulting process ID: 0x1374
Faulting application start time: 0x01d6d8ac5e2113d0
Faulting application path: F:\Steam\steamapps\common\MicrosoftFlightSimulator\FlightSimulator.exe
Faulting module path: F:\Steam\steamapps\common\MicrosoftFlightSimulator\FlightSimulator.exe
Report ID: 61fc82e0-4ed1-4bbd-8133-f33699580a84
Faulting package full name: 
Faulting package-relative application ID: 

And an information event

Fault bucket 1585757561155647880, type 4
Event Name: APPCRASH
Response: Not available
Cab Id: 0

Problem signature:
P1: AUDIODG.EXE
P2: 10.0.18362.1256
P3: dda6cbad
P4: ntdll.dll
P5: 10.0.18362.1171
P6: 103a4719
P7: c0000005
P8: 00000000000a01fd
P9: 
P10:

**Reply by Kirkles53** [reply]
Same problem for me as well when starting from the runway.  Have not tried starting at the gate yet.   Looks like it may be an audio issue?

**Comment by maxpull82** [comment]
dopo l'ultimo aggiornamento di msfs, anche a me crasha, esce direttamente dal simulatore togliendo l'a320 funziona forse qualche problema nella mod. speriamo risolvano.

**Reply by dellycowboy** [reply]
Also if you load a flight from the gate for both dev and stable 0.5 versions the plane seems very broken! 

APU won’t start, all of the 2nd fuel pumps for each fuel tank wont turn on. I tried Ctrl-E but the engines die during start up.

Think 1.12 has broken FBW, is this just me?

**Reply by dennis-koenig** [reply]
Same issue here. Using current stable version with the lates MSFS Update (1.12.13.0) broke APU and 2nd fuel pumps.

**Comment by Mickg56** [comment]
Hi dellycowboy I tried starting flight from gate, like you said, and that worked. No ctd.  Also, last10-12 flights VFR map freezes on second to last waypoint and display in cockpit(can't remember what its called) also freezes so makes it difficult to know where you are interaction to landing..does this on both ILS & RNAV approaches..Sent from my Galaxy
-------- Original message --------From: dellycowboy <notifications@github.com> Date: 23/12/20  9:22 am  (GMT+10:00) To: flybywiresim/a32nx <a32nx@noreply.github.com> Cc: Mickg56 <mickg0756@gmail.com>, Author <author@noreply.github.com> Subject: Re: [flybywiresim/a32nx] 1.12.13.0 ctd (#2618) 
So I can recreate this crash with stable and dev versions.
It only happens when starting a flight from the runway, from a gate is unaffected.
As the flight is about to load, I hear sounds then the Captain starts to say "seats for take-off" and I get CTD with 2 Event Log Errors



Faulting application name: AUDIODG.EXE, version: 10.0.18362.1256, time stamp: 0xdda6cbad
Faulting module name: ntdll.dll, version: 10.0.18362.1171, time stamp: 0x103a4719
Exception code: 0xc0000005
Fault offset: 0x00000000000a01fd
Faulting process ID: 0x5b40
Faulting application start time: 0x01d6d8ac8750cf49
Faulting application path: C:\Windows\system32\AUDIODG.EXE
Faulting module path: C:\Windows\SYSTEM32\ntdll.dll
Report ID: 35dbb574-7f9d-42e9-b278-308e49ee40fb
Faulting package full name:
Faulting package-relative application ID:



Faulting application name: FlightSimulator.exe, version: 1.12.13.0, time stamp: 0x5fda3fe4
Faulting module name: FlightSimulator.exe, version: 1.12.13.0, time stamp: 0x5fda3fe4
Exception code: 0xc0000005
Fault offset: 0x000000000190870a
Faulting process ID: 0x1374
Faulting application start time: 0x01d6d8ac5e2113d0
Faulting application path: F:\Steam\steamapps\common\MicrosoftFlightSimulator\FlightSimulator.exe
Faulting module path: F:\Steam\steamapps\common\MicrosoftFlightSimulator\FlightSimulator.exe
Report ID: 61fc82e0-4ed1-4bbd-8133-f33699580a84
Faulting package full name:
Faulting package-relative application ID:
And an information event
Fault bucket 1585757561155647880, type 4
Event Name: APPCRASH
Response: Not available
Cab Id: 0
Problem signature:
P1: AUDIODG.EXE
P2: 10.0.18362.1256
P3: dda6cbad
P4: ntdll.dll
P5: 10.0.18362.1171
P6: 103a4719
P7: c0000005
P8: 00000000000a01fd
P9:
P10:

**Comment by Mickg56** [comment]
It does for dellycowboy...I started from gate and it was ok...Sent from my Galaxy
-------- Original message --------From: Kirkles53 <notifications@github.com> Date: 23/12/20  4:41 pm  (GMT+10:00) To: flybywiresim/a32nx <a32nx@noreply.github.com> Cc: Mickg56 <mickg0756@gmail.com>, Author <author@noreply.github.com> Subject: Re: [flybywiresim/a32nx] 1.12.13.0 ctd (#2618) 
Same problem for me as well when starting from the runway.  Have not tried starting at the gate yet.   Looks like it may be an audio issue?
